File: client/src/app/_dtos/admin/userHistoryDto.ts
import { IUserHistoryItem } from "../../models/admin/userHistoryItem";

export interface IUserHistoryDto
{
     id: number;
     gender: string;
     userHistoryHeaderId?: number;
     checked?: boolean;
     source: string;
     categoryRef: string;
     categoryName: string;
     resumeId: string;
     name: string;
     personId?: number;
     personType: string;
     emailId: string;
     alternateEmailId: string;
     mobileNo:string;
     alternatePhoneNo: string;
     applicationNo?: number;
     createdOn: Date;
     concluded?: boolean;
     concludedByName: string;
     status: string;
     userName: string;
     errorMessage: string;
     userHistoryItems: IUserHistoryItem[];
}

File: api/DTOs/Admin/UserHistoryDto.cs
namespace api.DTOs.Admin
{
    public class CallRecddddordDto
    {
        public int Id {get; set;}
        public int CandidateId { get; set; }
        public int ApplicationNo {get; set;}
        public string ResumeId { get; set; }  
        public string Gender {get; set;}
        public string Source {get; set;}
        public string CategoryRef { get; set; }
        public string CategoryName { get; set; }
        public string CandidateName {get; set;}
        public string EmailId {get; set;}
        public string MobileNo { get; set; }
        public string AlternatePhoneNo { get; set; }
        
        public DateTime CreatedOn {get; set;}
        public DateTime ConcludedOn { get; set; }
        public string Status { get; set; }
        public string UserName { get; set; }
        public ICollection<CallRecordItemDto> CallRecordItems { get; set; }
    }
}